library/src/main/java/com/getstream/sdk/chat/adapter/MediaAttachmentSelectedAdapter.java

```diff
@@ -75,6 +75,13 @@ public void bind(Attachment attachment, final OnItemClickListener listener) {
                     .solidColor(Color.BLACK)
                     .cornerRadii(cornerRadius, cornerRadius, cornerRadius, cornerRadius)
                     .build());
+
+            binding.ivMask.setShape(context, new DrawableBuilder()
```

Comment:
Length preview is still wrong (video is a couple minutes)

![image](https://user-images.githubusercontent.com/88735/66920307-4e518e00-f023-11e9-819b-76717e5b8b8e.png)

Reply:
I can't reproduce the issue in my side.
Can you please send me the problem video?